data/heliarch intro.txt

```diff
@@ -0,0 +1,1568 @@
+			`	It seems Tugroob found some evidence of some smuggled weapons passing through here, but nothing major. They get back in their bunks, saying you may depart now.`
+				accept
+			label trees2
+			`	You suggest looking for something unusual near the plant life you spotted, while the rain is still fairly tame, and he follows you. You spend a few minutes looking around the trees, but find nothing suspicious.`
+			`	While passing by another set of bushes, he notices that some of them are awkwardly tilted. Upon further inspection, he tells you that they appear to have been removed, and then forcibly put on the ground again, noting some odd, wavy patterns of the small rocks around the roots. The rain gets even heavier.`
+			choice
+				`	(Dig out the bushes.)`
+					goto bushending
+				`	(Head back to the open warehouses and take shelter there.)`
+					goto beetledigger
+			label beetledigger
+			`	Though you can barely hear each other in the rain, Kirrie insists in staying there and digging through the ground for a while, to see if he finds anything.`
+			`	You walk back to the empty streets, taking care to not slip on the damp soil, and enter one of the warehouses Kirrie had previously opened. After a few minutes, he walks in, saying he dug out many, but couldn't find anything under the sets of bushes.`
+			`	Once the weather settles, near nightfall, he gets notified by Iplora that she and Tugroob are already back at the ship, so you two decide to head back as well.`
+			`	It seems Tugroob found some evidence of some smuggled weapons passing through here, but nothing major. They get back in their bunks, saying you may depart now.`
+				accept
+			label lonebeetle
+			`	He leaves too, and you're on your own in the ship for a couple hours. Eventually the get back, and it seems only Tugroob found something, mentioning some evidence of smuggled weapons passing through the planet.`
+			`	The three agents get back in their bunks, and say you may depart now.`
+
+
+
+mission "HI2-SF"
+	name "Investigate Stronghold of Flugbu"
+	description "Transport the Heliarch trio here so that they can investigate signs of Resistance activity."
+	to offer
+		has "Heliarch Investigation 1: done"
+	source "Ring of Friendship"
+	destination "Stronghold of Flugbu"
+	to fail
+		or
+			has "Heliarch Investigation 2: declined"
+			has "Heliarch Investigation 2: aborted"
+			has "Heliarch Investigation 2 :failed"
+	on complete
+		conversation
+			`While you doubt anyone opposing the Heliarchs would find much success in this massive compound, it seems the planet holds enough resources, ships and a valuable positioning that it was still marked for investigation.`
+			`	The three agents say you cannot go to most of the parts here they will be headed to, so you stay inside your ship. They return right as the sun is setting, and say they haven't found anything.`
```

Comment:
`	The three agents informs you that you're not allowed to enter most of the compound they will be headed to, leaving you behind to stay inside your ship. They return right as the sun is setting, revealing that they've found no signs of the resistance.`